static/css/new-case-studies.css

```diff
@@ -126,6 +126,13 @@ h4[id]:before {
   margin: 0 auto;
 }
 
+.content .quote-text p {
```

Comment:
quote-text is class attribute of div

Reply:
Yes, the changes is necessary since youtube shortcode produces div, and you can't wrap div inside p when nesting Hugo shortcodes.